lib/realtime/postgres_cdc.ex

```diff
@@ -18,15 +23,18 @@ defmodule Realtime.PostgresCdc do
     apply(module, :handle_subscribe, [pg_change_params, tenant, metadata])
   end
 
+  @spec stop(module, Tenant.t(), pos_integer) :: :ok
   def stop(module, tenant, timeout \\ @timeout) do
-    apply(module, :handle_stop, [tenant, timeout])
+    with {:ok, node} <- Realtime.Nodes.get_node_for_tenant(tenant) do
+      Rpc.enhanced_call(node, module, :handle_stop, [tenant.external_id, timeout])
```

Comment:
why do we need this change?

Reply:
will remove, killing via pid alone should be enough.

my thought was that the owner node should be the one killing the pid but that doesn't bring much benefit